lib/Config.js

```diff
@@ -313,7 +313,7 @@ class Config extends EventEmitter {
         return {
             transport: destConfig.sites?.[site]?.transport || destConfig.transport,
             auth: destConfig.sites?.[site]?.auth || destConfig.auth,
-            bootstrapList: this.bootstrapList,
+            replicationEndpoint: this.bootstrapList.find(endpoint => endpoint.site === site),
```

Comment:
is there anything more useful in bootstrapList ?
* `servers` may be an array (or sometimes not? then that case should be handled)
* the `site` field is nto relvant once we build a specific sites destination endpoint

```suggestion
            endpoints: this.bootstrapList.find(endpoint => endpoint.site === site).servers,
```

Reply:
There are other useful fields yes. For CRR: `servers`, `default`, `echo`. And for MultipleBackends: `type`